app/src/main/java/com/courseproject/tindar/entities/MessageInterface.java

```diff
@@ -0,0 +1,26 @@
+package com.courseproject.tindar.entities;
+
+import java.sql.Timestamp;
+
+/**
+ * Why does this exist? 
+ */
+
+public interface MessageInterface {
```

Comment:
Do we need this interface since this just has getters?

Reply:
This is purely here because the TA said we should have an interface. I'm also being a little less picky about more backend logic on this PR because I'm planning to fix and flesh it out in a separate PR. We can bring it up to the TA later; it shouldn't be a super difficult fix.